### PR TITLE
Update chronicle version to 2024.09.0

### PR DESCRIPTION
**Pull request by matt-urbina**
### Background
Part of the Chronicle release described here: https://github.com/rstudio/chronicle/issues/1067. The Chronicle team has made a new code release and we're updating our helm to match.

### Summary
Updates Chronicle server values
Update our docs
Update the Connect/Workbench/PPM CI files to match these Chronicle updates
This requires bumping up the chart versions for Connect/Workbench/PPM as well

### CODE REVIEW

**Review by markrtucker — COMMENTED** [review]
LGTM - we obviously can't merge this until we promote the release

**Review by t-margheim — APPROVED** [review]

File: charts/rstudio-connect/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Posit Connect
 
-![Version: 0.7.6](https://img.shields.io/badge/Version-0.7.6-informational?style=flat-square) ![AppVersion: 2024.08.0](https://img.shields.io/badge/AppVersion-2024.08.0-informational?style=flat-square)
+![Version: 0.7.7](https://img.shields.io/badge/Version-0.7.7-informational?style=flat-square) ![AppVersion: 2024.08.0](https://img.shields.io/badge/AppVersion-2024.08.0-informational?style=flat-square)
```
Comment (markrtucker) [inline charts/rstudio-connect/README.md:3]: Why is this not `0.7.8`? I would expect this to have the latest version.

Comment (matt-urbina) [reply charts/rstudio-connect/README.md:3]: It should, there was a bit of messiness with the merge, Im trying to work through that

File: charts/rstudio-connect/NEWS.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
 # Changelog
+## 0.7.8
+
+- Bump Chronicle Agent to version 2024.09.0
+
+## 0.7.7
+
+- Add helm values for `pod.hostAliases` and `launcher.templateValues.pod.hostAliases`
+- Add helm values for `launcher.defaultInitContainer.resources`
```
Comment (markrtucker) [inline charts/rstudio-connect/NEWS.md:9]: I'm a bit confused here - this change is already [on main](https://github.com/rstudio/helm/blob/main/charts/rstudio-connect/NEWS.md). Why is this included in the PR?

File: charts/rstudio-connect/README.md (hunk 1 above)
Comment (matt-urbina) [reply charts/rstudio-connect/README.md:3]: I don't understand why the diff here isn't showing main but the diff from the original commits. I used vscode to do the merge and am not liking the result. Debating blowing this away and getting a clean Branch & PR

**Comment by matt-urbina** [comment]
Closing this PR and creating a new one, some issues when merging using VS Code UI
